libsolidity/analysis/TypeChecker.cpp

```diff
@@ -1146,6 +1146,11 @@ bool TypeChecker::visit(FunctionCall const& _functionCall)
 	else
 		_functionCall.annotation().type = make_shared<TupleType>(functionType->returnParameterTypes());
 
+	if (functionType->bound() && !functionType->hasSelfType()) {
```

Comment:
I think that the member lookup should fail in the first place, or do I have a example case in mind? Could you add a test case for this?